src/pages/form-validation/list/index.tsx

```diff
@@ -0,0 +1,125 @@
+            margin: 0,
+            borderRadius: 0,
+            boxShadow: 'none',
+        },
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: '0 16px',
+    },
+    textFieldRoot: {
+        [theme.breakpoints.down('xs')]: {
+            width: 138,
+        },
+    },
+    skinnyInput: {
+        paddingTop: 11,
+    },
+}));
+
+export const ListFormValidation = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const [ip, setIp] = useState('10.0.0.1');
+    const dispatch = useDispatch();
+
+    const onIpChange: OnChangeHandler = useCallback((event) => {
+        setIp(event.target.value);
+    }, []);
+
+    return (
+        <>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'} classes={{ root: classes.appbarRoot }}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <Menu />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        In a List
+                    </Typography>
+                </Toolbar>
+            </AppBar>
+
+            <div className={classes.containerWrapper}>
+                <Card className={classes.container} elevation={4}>
+                    <List disablePadding style={{ width: '100%' }}>
+                        <InfoListItem
+                            icon={<Dns />}
+                            divider={'partial'}
+                            title={'IP Address'}
+                            rightComponent={
+                                <TextField
+                                    id="filled-basic"
+                                    classes={{
+                                        root: classes.textFieldRoot,
+                                    }}
+                                    value={ip}
+                                    InputProps={{
+                                        classes: {
+                                            input: classes.skinnyInput,
+                                        },
+                                    }}
+                                    onChange={onIpChange}
+                                    variant="filled"
+                                />
+                            }
+                        ></InfoListItem>
+
+                        <InfoListItem
+                            icon={<Timeline />}
```

Comment:
I'd still like to see that `insights` icon if it's available.

Reply:
Yeah the insights icon has been an issue, the current version of material-ui/icons we're using in this repo doesn't have the Insights icon available.   I didn't want to update the version just to get this icon so I think Timeline is a good compromise.